content/en/docs/reference/glossary/garbage-collection.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Garbage Collection
+id: garbage-collection
+date: 2021-07-07
+full_link: /docs/concepts/workloads/controllers/garbage-collection/
+short_description: >
+  A collective term for the various mechanisms Kubernetes uses to clean up cluster
+  resources.
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+ Garbage collection is a collective term for the various mechanisms Kubernetes uses to clean up
+ cluster resources. 
+
+<!--more-->
+
+Kubernetes uses garbage collection to clean up resources like [unused containers and images](/docs/concepts/workloads/controllers/garbage-collection/#containers-images),
+[failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection),
+deleted nodes, [objects without owner references](/docs/concepts/overview/working-with-objects/owners-dependents/), expired resources, and [completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/).
```

Comment:
How about:
```suggestion
[completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/),
resources that have expired or failed, and objects that
[belong to another object which is being deleted](/docs/concepts/overview/working-with-objects/owners-dependents/).
```
?
What do you think?

Reply:
Played with it a bit but yes I like the change.